src/openms/source/CHEMISTRY/ElementDB.cpp

```diff
@@ -139,45 +139,1652 @@ namespace OpenMS
     return smallest_weight;
   }
 
-  void ElementDB::storeElement_(const UInt an, const String& name, const String& symbol, const Map<UInt, double>& Z_to_abundancy, const Map<UInt, double>& Z_to_mass)
+  void ElementDB::storeElements()
+  {	
+
+    Map<UInt, double> hydrogen_abundance;
```

Comment:
I think you could just write:
std::map<UInt, double> hydrogen_abundance = { 
  { 1, 0.999885 }, 
  { 2, 1.15e-04 },
  { 3, 0.0 } 
};

Reply:
probably need to write the integer literal as: 1U to indicate unsigned int

Reply:
Yes, this is what I wanted to do in the next step cause it quite shortens the code.